app/src/main/java/at/ac/fhcampuswien/budget_fox/view_models/StatisticsViewModel.kt

```diff
@@ -20,9 +21,22 @@ class StatisticsViewModel : ViewModel() {
     val expenses: StateFlow<Map<Int, Double>>
         get() = _expenses.asStateFlow()
 
+    private var _selectedYear = MutableStateFlow(Calendar.getInstance().get(Calendar.YEAR))
+    val selectedYear: StateFlow<Int>
+        get() = _selectedYear.asStateFlow()
+
+    init {
```

Comment:
Aufpassen! Hatten wir in einer alten Revision schon, da hatten wir hin und wieder Race-Conditions!